mlflow/pyfunc/backend.py

```diff
@@ -152,7 +153,9 @@ def serve(
         local_path = _download_artifact_from_uri(model_uri)
 
         server_implementation = mlserver if enable_mlserver else scoring_server
-        command, command_env = server_implementation.get_cmd(local_path, port, host, self._nworkers)
+        command, command_env = server_implementation.get_cmd(  # pylint: disable-msg=E1121
```

Comment:
```suggestion
        command, command_env = server_implementation.get_cmd(  # pylint: disable-msg=E1121
```

Do we need this pylint comment?

Reply:
Removed it, thanks!